augur/parse.py

```diff
@@ -5,7 +5,7 @@
 
 from .io import open_file, read_sequences, write_sequences
 
-forbidden_chactacters = str.maketrans(
+forbidden_characters = str.maketrans(
```

Comment:
So as not to break anything, one could add `forbidden_chactacters = forbidden_characters` after this statement just in case someone uses them. But I doubt it.

Reply:
Good point! This isn't part of our formal API (which is still not yet documented, so also not really formal), so I would not consider this a breaking change. If this were part of our API, our policy would be to make a new major release to denote the backward incompatibility, instead of maintaining the spelling mistake in the API.